spark2/src/test/java/org/apache/iceberg/examples/SchemaEvolutionTest.java

```diff
@@ -120,13 +161,25 @@ public void renameColumn() {
 
     results.createOrReplaceTempView("table");
     spark.sql("select * from table").show();
+    List<String> fields = Arrays.asList(spark.sql("select * from table").schema().names());
+    assert (fields.contains("writer"));
+    assert (!fields.contains("author"));
```

Comment:
This are good assertions to have, but they should be JUnit assertions and not `assert`.

Reply:
done